### DISCUSSION TITLE
Project 5: individual trees or counts??

### DISCUSSION THREAD

**Discussion by Rawizard** [IBAHCM/RPiR#338] [Archive] [answered]
Hi @soniamitchell & @richardreeve,

I have a question about project 5 I was hoping you could clarify.

When you say an "individual" are you referring to exactly that (i.e one tree from one species) or are you referring to an individual count?

I really hope its the later. 

Cheers,

Jamie

**Comment by soniamitchell** [comment, marked as answer]
Hi @Rawizard, individual trees 👍

**Reply by richardreeve** [reply]
I have to admit I don’t really understand the question - I’m not sure what a sample of 10 “counts” would mean - but I agree with Sonia that we mean trees.

**Reply by Rawizard** [reply]
Thanks Sonia, yeah Richard I don't think it makes any sense so ignore me, realised I was confusing myself as my original function was wrong. Anyhow thanks!
